OnProfileUpdate/index.ts

```diff
@@ -0,0 +1,53 @@
+import { Context } from "@azure/functions";
+import { pipe } from "fp-ts/lib/function";
+import { TableClient, AzureNamedKeyCredential } from "@azure/data-tables";
+import { DataTableProfileEmailsRepository } from "@pagopa/io-functions-commons/dist/src/utils/unique_email_enforcement/storage";
+import {
+  PROFILE_COLLECTION_NAME,
+  ProfileModel
+} from "@pagopa/io-functions-commons/dist/src/models/profile";
+import { CosmosClient } from "@azure/cosmos";
+import { getConfigOrThrow } from "../utils/config";
+import { handler } from "./handler";
+
+const config = getConfigOrThrow();
+
+const credential = new AzureNamedKeyCredential(
+  config.AZURE_STORAGE_ACCOUNT_NAME,
+  config.AZURE_STORAGE_ACCOUNT_KEY
+);
+
+const tableClient = new TableClient(

```

Comment:
these clients must be singletons. use cosmos client from `../utils/cosmosdb.ts` and put the table client into some shared module instead within the function code itself.